packages/components/fields/creatable-select-field/src/creatable-select-field.tsx

```diff
@@ -0,0 +1,453 @@
+  containerId?: ReactSelectCreatableProps['id'];
+  /**
+   * Is the select value clearable
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isClearable?: ReactSelectCreatableProps['isClearable'];
+  /**
+   * Is the select disabled
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isDisabled?: ReactSelectCreatableProps['isDisabled'];
+  /**
+   * Is the select read-only
+   */
+  isReadOnly?: ConstrainBooleanParameters;
+  /**
+   * Override the built-in logic to detect whether an option is disabled
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isOptionDisabled?: ReactSelectCreatableProps['isOptionDisabled'];
+  /**
+   * Support multiple selected options
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isMulti?: ReactSelectCreatableProps['isMulti'];
+  /**
+   * Whether to enable search functionality
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  isSearchable?: ReactSelectCreatableProps['isSearchable'];
+  /**
+   * Indicates the input field has an error
+   */
+  hasWarning?: boolean;
+  /**
+   * Maximum height of the menu before scrolling
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  maxMenuHeight?: ReactSelectCreatableProps['maxMenuHeight'];
+  /**
+   * 	Dom element to portal the select menu to
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  menuPortalTarget?: ReactSelectCreatableProps['menuPortalTarget'];
+  /**
+   * z-index value for the menu portal
+   */
+  menuPortalZIndex?: number;
+  /**
+   * whether the menu should block scroll while open
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  menuShouldBlockScroll?: ReactSelectCreatableProps['menuShouldBlockScroll'];
+  /**
+   * 	Name of the HTML Input (optional - without this, no input will be rendered)
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  name?: ReactSelectCreatableProps['name'];
+  /**
+   * Can be used to render a custom value when there are no options (either because of no search results, or all options have been used, or there were none in the first place).
+   * <br/>
+   * Gets called with `{ inputValue: String }`. `inputValue` will be an empty string when no search text is present.
+   * <br>
+   * [Props from React select was used](https://react-select.com/props#creatable-props)
+   */
+  noOptionsMessage?: ReactSelectCreatableProps['noOptionsMessage'];
+  /**
+   * Handle blur events on the control
+   */
+  onBlur?: (event: TEvent) => void;
+  /**
+   * Called with a fake event when value changes.
+   * <br />
```

Comment:
nit: We have `<br>`, `<br/>` and `<br />` across the props.
Maybe we could standarize it.